load/BlimpLanguageReader.py

```diff
@@ -84,3 +85,26 @@ def getGospelsAndApostolicHistoryBooks(self):
     # Get the list of books for the Covenant group
     def getCovenantBookId(self):
         return "COV"
+
+    def get_existence_files_by_fileset_id(self, fileset_id: str) -> List[str]:
```

Comment:
rename to get_existing_files_by_fileset_id

Reply:
@bradflood , it has been updated as you can view here: https://github.com/faithcomesbyhearing/dbp-etl/pull/227/files#diff-95ea1b21a4bf737e4e1d09e2a50cb69a1b5065a90b56f721a2e79135e2077e05R89